pegasus/sites.v3/code.org/public/ops-dashboard/index.haml

```diff
@@ -143,7 +144,7 @@ title: "Ops Dashboard"
       var Attendance = $resource('/dashboardapi/attendance/workshop/:workshopid', {}, {
         // default methods: see https://code.angularjs.org/1.2.21/docs/api/ngResource/service/$resource
         //  'get':    {method:'GET'},
-        //  'save':   {method:'POST'},
+        //  'save':   {method:'POST', url:'/dashboardapi/segments/:segmentid/attendance'},
```

Comment:
for that matter, we're not using the attendanceService now, I think? Are you keeping it around because you want to add saving an individual attendance (as opposed to only saving the entire thing at once?).